### DISCUSSION TITLE
Tags in the abs metadata file???

### DISCUSSION THREAD

**Discussion by Hallo951** [advplyr/audiobookshelf#1325] [Q&A]
Why are no tags stored in the abs metadata file? Wouldn't that make sense?

**Comment by advplyr** [comment]
There is a request for this https://github.com/advplyr/audiobookshelf/issues/772

**Comment by hobesman** [comment]
Tags are kind of in limbo right now. Originally they were intended as something personal. Like marking a book with your own remarks or flags. "Ok for the kids" or whatever. But later they were merged with audible tags, which is almost like an expansion of genre. Tagging it with elements of the content, like "werewolves" or "space travel". And it still isn't decided what to do about the dual purposes. Separate them out? Stop using it for one purpose and keep the other?

But I think either way it would make sense to keep it in the metadata file. If I go through and tag a bunch of books to be approved for the kids, and then my library gets corrupted or something, I want to be able to restore all of that information, whichever purpose I'm using.

**Comment by Hallo951** [comment]
Yes, that's exactly how I see it too. That's why I asked... thank you for your detailed answer.
